.travis.yml

```diff
@@ -1,57 +1,170 @@
+# This Travis job script has been generated by a script via
+#
+#   haskell-ci 'vault.cabal'
+#
+# For more information, see https://github.com/haskell-CI/haskell-ci
+#
+# version: 0.9.20200321
+#
+
 # I do not wish to receive email notifications
 notifications:
   email: false
```

Comment:
I included this bit manually since I didn't find a way to configure it in `haskell-ci`: https://github.com/haskell-CI/haskell-ci/issues/369